docs/user-guide/update-demo/index.md.orig

```diff
@@ -0,0 +1,107 @@
+---

```

Comment:
I left a copy of the original in there just in case as part of the docs migration. Once we finish clearing out the user-guide directory, it will be removed. If everything looks okay, you can remove it now if you want.